src/server/search/search_family_test.cc

```diff
@@ -3573,4 +3574,81 @@ TEST_F(SearchFamilyTest, KnnHnsw) {
   EXPECT_THAT(resp, AreDocIds("doc3", "doc1"));
 }
 
+TEST_F(SearchFamilyTest, ParseCSSResponse) {
+  using Fields = std::map<std::string, std::string>;
```

Comment:
nit: seems like not the best place for such low-level test. you can introduce a new test under src/redis and test this there.